imx.mk

```diff
@@ -136,7 +137,9 @@ $(FIRMWARE_PATH)/.unpacked: $(FIRMWARE_PATH)/$(FIRMWARE_BIN)
 	(cd $(FIRMWARE_PATH) && \
 	 chmod 711 $(FIRMWARE_BIN) && ./$(FIRMWARE_BIN) --auto-accept)
 	touch $(FIRMWARE_PATH)/.unpacked
-	
+	cp $(FIRMWARE_PATH)/$(FIRMWARE_VERSION)/firmware/ddr/synopsys/lpddr4_pmu_train_*.bin \
+		$(U-BOOT_PATH)
```

Comment:
(Commit " imx.mk: copy the ddr firmwares and tf-a bl31 binray to U-Boot")
This is fragile. If `u-boot` is cleaned (removed and cloned again, or `git clean -f`), then `make u-boot` won't be able to build it again. I would prefer to have this in the u-boot build recipe.

Reply:
ack, will do.